server/s3_copy_object_action.cc

```diff
@@ -47,6 +47,8 @@ S3CopyObjectAction::S3CopyObjectAction(
   old_object_oid = {0ULL, 0ULL};
   old_layout_id = -1;
   new_object_oid = {0ULL, 0ULL};
+  source_bucket_name = "";
```

Comment:
It's redundant. Default constructor of std::string initialize it as empty string.

Reply:
Fixed.